evm_loader/lib/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+pub mod account_storage;
+pub mod commands;
+pub mod config;
+pub mod context;
+pub mod errors;
+pub mod event_listener;
+pub mod rpc;
+pub mod syscall_stubs;
+pub mod types;
+
+pub use config::Config;
+pub use context::Context;
+pub use errors::NeonCliError;
+
+pub type NeonCliResult = Result<serde_json::Value, NeonCliError>;
```

Comment:
We should get rid of NeonCliResult in the library. Let it return scalars/structures with solana/evm library errors, and CLI will wrap them to JSON and return NeonCliResult only in CLI functions.